readthedocs/projects/tasks/utils.py

```diff
@@ -237,8 +237,9 @@ def deprecated_config_file_used_notification():
             project.versions.filter(slug=project.default_version).only("id").first()
         )
         if version:
+            years_ago = timezone.now() - timezone.timedelta(days=365)  # 1 years ago
```

Comment:
The only weird thing is the set of projects will change each day. I wonder if we just pick June 1, 2022 as the cutoff?

Reply:
Done!